src/app/apps/lti/tests/test_lti_request.py

```diff
@@ -0,0 +1,157 @@
+"""Tests for the LTI request view."""
+
+
+import json
+import re
+from html import unescape
+from logging import Logger
+from unittest import mock
+from urllib.parse import urlencode
+
+from django.test import TestCase, override_settings
+from lti_toolbox.factories import LTIConsumerFactory, LTIPassportFactory
+
+from .utils import CONTENT_TYPE, sign_parameters
+
+# todo - discuss ending slash
+ROUTE = "/lti/test/"
```

Comment:
Maybe we call it URL instead (to keep Django's semantic):

```suggestion
TARGET_URL_PATH = "/lti/test/"
```